src/smith/physics/tests/dynamic_solid_adjoint.cpp

```diff
@@ -422,14 +422,16 @@ TEST_F(SolidMechanicsSensitivityFixture, QuasiStaticShapeSensitivities)
 TEST_F(SolidMechanicsSensitivityFixture, WhenShapeSensitivitiesCalledTwice_GetSameObjectiveAndGradient)
 {
   auto solid_solver = createNonlinearSolidMechanicsSolver();
-  auto [qoi1, _, __, shape_sensitivity1] = computeSolidMechanicsQoiSensitivities(*solid_solver, tsInfo);
+  auto [qoi1, shape_unused1, shape_unused2, shape_sensitivity1] =
```

Comment:
Was this needed?

Reply:
yes. i was getting an error about duplicated use of `____` - or something like that.

Reply:
Sound good.